protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -67,6 +73,13 @@ async fn test_movement_client_build_and_fund_accounts() -> Result<(), anyhow::Er
 			.await
 			.context("Failed to get Bob's account balance")?
 	);
+	println!(
```

Comment:
instead of printing why can't we assert on the value in the balance perhaps?

Reply:
Updated with an assertion.